app/src/main/java/uk/gov/onelogin/network/auth/response/TokenResponse.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.onelogin.network.auth.response
+
+import androidx.annotation.Keep
+import com.google.gson.annotations.SerializedName
+import kotlinx.serialization.Serializable
+
+@Keep
+@Serializable
+data class TokenResponse(
+    @SerializedName("access_token")
```

Comment:
`@SerializedName` is the property name from the JSON object when serialized:

```
{
  access_token: "some token"
}
```

Internally that value is stored as `TokenRespose.access`